pipelines/A14/queries.sql

```diff
@@ -56,34 +56,35 @@ SELECT T.total_full_time_employment_of_tourism
 FROM tourism_economy_by_canton as T
 JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid
 WHERE S.country=TRUE
-    AND S.name_de = "Schweiz"
+    AND S.name_de = 'Schweiz'
```

Comment:
```suggestion
    AND S.name = 'Switzerland'
```
Should we not use `S.name` to be consistent? (like in A10)